server/config/raw/global_cfg.go

```diff
@@ -26,22 +26,22 @@ type GlobalCfg struct {
 	Persistence          Persistence          `yaml:"persistence" json:"persistence"`
 	RevisionSetter       RevisionSetter       `yaml:"revision_setter" json:"revision_setter"`
 	Admin                Admin                `yaml:"admin" json:"admin"`
-	TerraformAdminMode   TerraformAdminMode   `yaml:"terraform_admin_mode" json:"terraform_admin_mode"`
+	AdminMode            AdminMode            `yaml:"admin_mode" json:"admin_mode"`
```

Comment:
Ah I see what you mean here, that's definitely confusing term. Maybe we can do something else, another suggestion I can throw out is `Manual` but maybe there's a better name here.